### ISSUE TITLE
Documentation - Include Guidelines on Using Sail to load RDF data files

### ISSUE THREAD

**Issue by darrenleeweber** [JanusGraph/janusgraph#191]
How are RDF data files loaded into JanusGraph?  Is there any way to stand-up a SPARQL endpoint on a JanusGraph stack?  Please see related discussion referencing Sail at
- https://groups.google.com/forum/#!topic/gremlin-users/nIE6uaSck8g

**Comment by twilmes** [comment]
Per the discussion in #190, there is not a TinkerPop 3 compatible Sail implementation.

**Comment by darrenleeweber** [comment]
Could the documentation include a short note that RDF/Sail is not supported?  That would, at least, resolve the question for anyone trying to find this information in the documentation.

**Comment by mbrukman** [comment]
Good point, @darrenleeweber. Please feel free to submit a PR if you're up for it, but note that this project requires a CLA to be submitted first..

**Comment by kyprifog** [comment]
Does anyone have an update on the status of this?

**Comment by archenroot** [comment]
Also interested, RDF format is emerging and RDF4J performance while better than Jena is still poor and I look for some high performance tripple store. Thx guys for update.

**Comment by charbull** [comment]
@archenroot @kyprifog @darrenleeweber any updates regarding this topic? I am also interested in teh RDF layer.
